src/scenes/Jarjestajat/Jarjestaja/Hakemukset/Muutospyynto/components/Yhteenveto/YhteenvetoLiitteet.js

```diff
@@ -0,0 +1,85 @@
+import React, { useEffect, useMemo } from "react";
+import ExpandableRowRoot from "../../../../../../../components/02-organisms/ExpandableRowRoot";
+import { injectIntl } from "react-intl";
+import PropTypes from "prop-types";
+import * as R from "ramda";
+
+const YhteenvetoLiitteet = React.memo(props => {
+  const { onStateUpdate, sectionId } = props;
+
+  const getCategories = useMemo(() => {
+    return () => {
+      let structure = null;
+
+      structure = [
+        {
+          anchor: "hakemuksenliitteet",
+          components: [
+            {
+              name: "StatusTextRow",
+              styleClasses: ["w-full"],
+              properties: {
+                title:
+                  "Liittäkää asiakirja tai asiakirjat, joista ilmenee hakemuksen hyväksyntä tai hyväksyjän päätösvalta (esim. hyväksyjän allekirjoitusoikeus ja päättävän elimen kokouksen pöytäkirjanote). Liitteen koko saa olla korkeintaan 25 MB ja tyypiltään pdf, word, excel, jpeg tai gif. Muistakaa merkitä salassa pidettävät liitteet."
+              }
```

Comment:
Could you use the title property of the category instead? The structure would be more simple.

Reply:
After this is supported with normal text also.